argo-app-of-apps.yaml

```diff
@@ -19,7 +19,7 @@ applications:
     enabled: true
     source: https://github.com/rht-labs/ubiquitous-journey.git
     source_path: ubiquitous-journey/
-    source_ref: master
+    source_ref: workshopcaio
```

Comment:
```suggestion
    source_ref: master
```

Reply:
yup, that's what I am suggesting also 🙃 Your commit has `source_ref: workshopcaio`